dsync.go

```diff
@@ -41,12 +42,10 @@ type Dsync struct {
 
 // New - initializes a new dsync object with input rpcClnts.
 func New(rpcClnts []NetLocker, rpcOwnNode int) (*Dsync, error) {
-	if len(rpcClnts) < 4 {
-		return nil, errors.New("Dsync is not designed for less than 4 nodes")
+	if len(rpcClnts) < 3 {
```

Comment:
We also decided to add support for 2 nodes where for write quorum we need 2 nodes available always.

Reply:
Additionally we are also moving to maximum of 32 nodes not 16 we need to change that as well and associated tests.